src/components/Accordion/Item/TriggerButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from "../../Button";
+import { ChervonDownIcon } from "../../Icons";
+import { icon } from "../common.css";
+
+interface TriggerButtonProps {
```

Comment:
question: Shouldn't this be exported?
```suggestion
export interface TriggerButtonProps {
```